review/index.html

```diff
@@ -1,71 +1,71 @@
+        <link rel="preconnect" href="https://fonts.googleapis.com" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap" rel="stylesheet" />
+        <link href="https://fonts.googleapis.com/css?family=Material+Icons|Material+Icons+Outlined|Material+Icons+Two+Tone|Material+Icons+Round|Material+Icons+Sharp" rel="stylesheet" />
+    </head>
+
+    <body>
+        <header class="header">
+            <div class="header__background"></div>
+            <div class="header__content">
+                <div class="logo"></div>
+                <div class="toggle"></div>
+            </div>
+        </header>
+
+        <main>
+            <div class="container">
+                <section><span>section 1</span></section>
+                <section><span>section 2</span></section>
+                <section><span>section 3</span></section>
+                <section><span>section 4</span></section>
+            </div>
+        </main>
+
+        <div class="menu">
+            <div class="menu__wrapper">
+                <section class="menu__section menu__section--left">
+                    <div class="links">-- various links goes here --</div>
+                </section>
+                <section class="menu__section menu__section--right">
+                    <div class="carousel">
+                        <div class="carousel__list">
+                            <ul class="carousel__slider">
+                                <li class="carousel__list-item">-- item with different background here --</li>
+                                <li class="carousel__list-item">-- item with different background here --</li>
+                                <li class="carousel__list-item">-- item with different background here --</li>
+                                <li class="carousel__list-item">-- item with different background here --</li>
+                            </ul>
+                        </div>
+
+                        <div class="carousel__controls">
+                            <div class="carousel__button carousel__button--prev">
+                                <span class="material-icons-outlined"> expand_less </span>
+                            </div>
+                            <div class="carousel__button carousel__button--next">
+                                <span class="material-icons-outlined"> expand_more </span>
+                            </div>
+                        </div>
+
+                        <div class="carousel__indexes">
+                            <span class="carousel__index carousel__index--active"></span>
+                            <span class="carousel__index"></span>
+                            <span class="carousel__index"></span>
+                            <span class="carousel__index"></span>
```

Comment:
Essendo tutti elementi cliccabili l'elemento utilizzato dovrebbe essere button e non span. Andrebbero inoltre aggiunte delle aria-label per motivi di accessibilità. L'utilizzo della classe carousel__index--active è giusto in questo caso in quanto denota un particolare stato css dell'elemento